scripts/cluster_network.py

```diff
@@ -177,8 +178,9 @@ def weighting_for_country(n, x):
     w = g + l
 
     if w.max() == 0.0:
+        country = pycountry.countries.get(alpha_2=x.name[0])
         logger.warning(
-            f"Null weighting for buses of country {x.country.iloc[0]}: returned default uniform weighting"
+            f"Null weighting for buses of country {country.name if country else x.name[0]}: returned default uniform weighting"
```

Comment:
Great catch anton! :D

What about simply:
```suggestion
            f"Null weighting for buses of country {x.name[0]}: returned default uniform weighting"
```

As it is only for informing the user it would be easier and avoids the import
FYI, for other applications, in the helpers we have auxiliary functions to convert country codes to country names.

Reply:
Just did it for better usability - but I agree, lets just take the alpha 2 code! :)
Have adjusted it - I guess we are ready to merge?

Reply:
Yesss :)